resources/sass/app.scss

```diff
@@ -1,11 +1,32 @@
 @import url('https://fonts.googleapis.com/css2?family=Lato&family=Source+Serif+Pro&display=swap');
 
 $mobile-max-size: 720px;
+$color-base-0: #000;
```

Comment:
Ideally, I would import these from WiKit already, but since this is a side priority, I would definitely agree that we can take care of that later, in another task.